core/collection.ts

```diff
@@ -37,7 +37,6 @@ export default class Collection {
     return new Document({
       entry: this.#storage.get(name),
       fields: this.#fields,
-      isNew: true,
```

Comment:
The `isNew` variable is required because it will fail on creating new documents, because it will try to read an unexisting file.